stock_picking_completion_info/models/stock_picking.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, models, fields
+
+
+class PickingType(models.Model):
+
+    _inherit = 'stock.picking.type'
+
+    display_completion_info = fields.Boolean(
+        help='Inform operator of a completed order pick at processing and at'
+             ' completion'
+    )
+
+
+class StockPicking(models.Model):
+
+    _inherit = 'stock.picking'
+
+    completion_info = fields.Selection(
+        [
+            ('no', 'No'),
+            ('last_picking', 'Completion of this picking allows next pickings '
+                             'to be processed.'),
+            ('next_picking_ready', 'Next pickings are ready to be processed.')
+        ],
+        compute='_compute_completion_info')
+
+    @api.depends(
+        'picking_type_id.display_completion_info',
+        'move_lines.move_dest_ids.move_orig_ids.state'
+    )
+    def _compute_completion_info(self):
+        for picking in self:
+            if (
+                picking.state == 'draft'
+                or not picking.picking_type_id.display_completion_info
+            ):
+                picking.completion_info = 'no'
+                continue
+            # Depending moves are all the origin moves linked to the
+            # destination pickings' moves
+            depending_moves = picking.move_lines.mapped(
+                'move_dest_ids.picking_id.move_lines.move_orig_ids'
+            )
+            if all(
```

Comment:
not one `elif` ?

Reply:
I used `continue` to avoid having to to many `elif` and spare some indent.